BigQuery/composer.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "google/cloud-bigquery",
+    "description": "BigQuery Client for PHP",
+    "license": "Apache-2.0",
+    "minimum-stability": "stable",
+    "require": {
+        "google/cloud-core": "^1.14",
+        "ramsey/uuid": "~3"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^4.8|^5.0",
+        "squizlabs/php_codesniffer": "2.*",
+        "phpdocumentor/reflection": "^3.0",
```

Comment:
Do we need to require `phpdocumentor/reflection` & `erusev/parsedown`? This comment applies to many of the other composer files as well.